assets/css/custom.css

```
@@ -1896,4 +1905,566 @@ a.read-all-posts:hover {
+}
+.hamburger--slider .hamburger-inner:before {
+  top: .9rem;
+  transition-duration: .15s;
+  transition-property: transform,opacity;
+  transition-timing-function: ease;
+}
+
+.hamburger-inner, 
+.hamburger-inner:after, 
+.hamburger-inner:before {
+  background-color: #000;
+  border-radius: 0.25rem;
+  height: 0.25rem;
+  position: absolute;
+  transition-duration: .15s;
+  transition-property: transform;
+  transition-timing-function: ease;
+  width: 3.5rem;
+}
+.hamburger--slider .hamburger-inner:after {
+  top: 1.8rem;
+}
+.hamburger-inner:after {
+  bottom: -0.625rem;
+}
+
+.hamburger.is-active .hamburger-inner, 
+.hamburger.is-active .hamburger-inner:after, 
+.hamburger.is-active .hamburger-inner:before {
+  background-color: #000;
+}
+
+.hamburger--slider.is_active .hamburger-inner {
+  transform: translate3d(0,1.2rem,0) rotate(45deg);
+}
+
+.hamburger--slider.is_active .hamburger-inner:before {
+    opacity: 0;
+    transform: rotate(-45deg) translate3d(-0.6rem,-0.6rem,0);
+}
+
+.hamburger--slider.is_active .hamburger-inner:after {
+  transform: translate3d(0,-1.8rem,0) rotate(-90deg);
+}
+
+.new-header .main_menu {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+@media (max-width:880px) {
+  .new-header .main_menu  {
+    display: none;
+  }
+}
+
+.new-header .main_menu>li {
+  display: flex;
+  font-size: 1.2rem;
+  align-items: center;
+  margin: 0 0.75rem;
+  padding-bottom: 2rem;
+  position: relative;
+  transition: .25s;
+}
+
+@media screen and (min-width: 970px) {
+    .new-header .top-nav.main_menu>li {
+      font-size: 1.5rem;
+      margin: 0 0.6vw;
+  }
+}
+
+.new-header .main_menu>li>button {
+  background-color: #fff;
+  border-bottom: 0.250rem solid transparent;
+  color: #000;
+  display: inline-block;
+  font-size: 1.1rem;
+  font-weight: 500;
+  line-height: 18px;
+  line-height: 1.8;
+  padding: 0;
+  padding-bottom: 0.0625rem;
+  position: relative;
+  text-transform: uppercase;
+  transition: color .25s;
+}
+
+@media screen and (min-width: 1200px) {
+  .new-header .main_menu>li>button {
+    font-size: 1.5rem;
+  }
+}
+
+.new-header .main_menu>li.current_page>button, 
+.new-header .main_menu>li:hover>button {
+  border-bottom-color: #0dddd3;
+}
+
+.new-header ul {
+  list-style: none;
+}
+
+.new-header .main_menu .sub-menu {
+  background-color: #fff;
+  border-radius: 0.6rem;
+  bottom: 0;
+  box-shadow: 0.5rem 0.5rem 6rem rgba(0,0,0,.25);
+  left: 0;
+  opacity: 0;
+  padding: 1.2rem 0;
+  position: absolute;
+  transform: translateY(100%);
+  transition: .25s;
+  visibility: hidden;
+  max-width: max-content;
+  width: 300px;
+}
+@media screen and (min-width: 970px) {
+    .new-header .top-nav.main_menu .sub-menu {
+      max-width: max-content;
+      width: 400px;
+  }
+}
+.new-header .main_menu>li:hover .sub-menu {
+  margin: 0;
+  opacity: 1;
+  visibility: visible;
+  z-index: 12;
+}
+
+.new-header .main_menu>li:hover .sub-menu li {
```

```suggestion
.new-header .main_menu>li .sub-menu li {
```
I found the source of tiny jump as the menu disappears! :)